### ISSUE TITLE
Not install update

### ISSUE THREAD

**Issue by Martos** [cizia64/CrossMix-OS#101]
Hi, i'm trying to install update from 1.0.4 to 1.1.0. I placed the zip on the root of SDCARD and update te script _Firmware_Check.sh but the install not starting.

**Comment by cobaltgit** [comment]
This update method only works for 1.2.0 and up, the full release of 1.2.0 should be up in the next few days?

**Comment by Martos** [comment]
Oh thanks ! So to update to 1.2.0 i need to do a fresh install, right ?

**Comment by cobaltgit** [comment]
No, I updated from 1.0.0 to 1.2.0 prerelease just fine, it's just 1.0.0 to 1.1.0 that won't work with this method, no?
